src/views/home/postAd/SectionLink/SectionLink.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { HiPlusCircle } from "react-icons/hi";
+import { Link } from "react-router-dom";
+
+const SectionLink = () => {
+  return (
+    <Link
+      to="#"
+      className="bg-white border-2 border-dark-blue hover:bg-dark-blue hover:text-white px-5 py-3 rounded-md uppercase text-dark-blue font-medium flex justify-center items-center gap-2 duration-300"
+    >
+      <HiPlusCircle className="text-xl" />
+      post your ad
+    </Link>
+  );
+};
+
+export default SectionLink;
```

Comment:
As I have seen in other sections, very similar code for buttons is used over and over again, maybe you might want to create a global component and use it in all sections instead? I mean, it's just a suggestion, you don't have to do it if you don't want to.